cmd/assign/serviceaccount.go

```diff
@@ -186,9 +187,14 @@ listServiceAccountGroupMembers:
 				logrus.Debugf("Adding %s to Group", address)
 				_, err := api.AddMember(adminApi, serviceAccountGroupAddress, address)
 				if err != nil {
-					logrus.Error("An error occurred when adding an account. Retrying...", err)
-					time.Sleep(100 * time.Millisecond)
-					goto addMemberToGroup
+					if strings.Contains(err.Error(),"409: Member already exists., duplicate") {
```

Comment:
You can check if its from the type googleApiError and if it is then extract the error code. Tho if you dont want to I can do that also

Reply:
Thanks for the feedback. I found how you handled this in `group.go`. Would that method be preferred? Something like this. Doesn't tell the user what happened, but if it's not game breaking then I don't see why it must. 
```
...
"google.golang.org/api/googleapi"
...
if err, ok := err.(*googleapi.Error); ok { 
    if err.Code != 409 && err.Code != 404 { 
        logrus.Error("An error occurred when adding an account. Retrying...", err)
        time.Sleep(100 * time.Millisecond)
        goto addMemberToGroup
    } 
} else if !ok { 
    logrus.Panic(err) 
}
```

Reply:
```go
if gerr, ok := err.(*googleapi.Error); ok {
	if gerr.Code == 409 {
    	logrus.Info("Account already exists. Skipping.")
        time.Sleep(100 * time.Millisecond)
    } else {
        logrus.Error("An error occurred when adding an account. Retrying...", err)
        time.Sleep(100 * time.Millisecond)
    	goto addMemberToGroup
	}
} else if !ok {
	logrus.Error(err)
}
```
like this